.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+<!-- Thanks so much for your PR, your contribution is appreaciated! -->
+
+- [ ] Description is clear and explains issue / use-case resolved.
+- [ ] PR has tests / docs demo, and is linted.
+- [ ] Commit and PR title begins with [ComponentName], and are in imperative form.
```

Comment:
Mmm... I could link here: http://chris.beams.io/posts/git-commit/, but I was trying to keep this template a bit more sparse, since the issues one got a bit wordy! :smile: 

If you can come up with a good example in as few words as possible, I son't see why not. :+1: